lobby/src/main/java/org/triplea/lobby/server/db/UserController.java

```diff
@@ -140,15 +140,16 @@ public String getUserEmailByName(final String username) {
 
   @Override
   public boolean isAdmin(final String username) {
-    final String sql = "select admin from lobby_user where username = ?";
+    final String sql = "select role from lobby_user where username = ?";
     try (Connection con = connection.get();
         PreparedStatement ps = con.prepareStatement(sql)) {
       ps.setString(1, username);
       try (ResultSet rs = ps.executeQuery()) {
         if (!rs.next()) {
           return false;
         }
-        return rs.getBoolean("admin");
+        final String role = rs.getString("role");
+        return "ADMIN".equals(role) || "MODERATOR".equals(role);
```

Comment:
I'm not sure about the compile-time dependencies, but it would be nice if those strings were stored in single global constants instead of hardcoding them here.

Reply:
It was a pretty late change to push 'UserRole' to the DB level, originally these constants were not available to 'lobby'. In the end I did push 'UserRole' from 'http-server' to 'lobby-db', so the constants can be used here now for sure. 

Updated in: d2d1ce369